browser-api/src/main/java/com/duckduckgo/app/accessibility/AccessibilityScreens.kt

```diff
@@ -22,3 +22,8 @@ import com.duckduckgo.navigation.api.GlobalActivityStarter.ActivityParams
  * Use this model to launch the Accessibility screen
  */
 object AccessibilityScreenNoParams : ActivityParams
+
+/**
+ * Use this model to launch the Accessibility screen with a highlighted item
+ */
+data class AccessibilityScreenParams(val highlightedItem: String) : ActivityParams
```

Comment:
I'd change to the following
```suggestion
sealed class AccessibilityScreens : ActivityParams {
    /**
     * Use this model to launch the Accessibility screen
     */
    data object Default : AccessibilityScreens() {
        private fun readResolve(): Any = Default
    }

    /**
     * Use this model to launch the Accessibility screen with a highlighted item
     */
    data class HighlightedItem(val highlightedItem: String) : AccessibilityScreens()
}
```

Reply:
Updated in: https://github.com/duckduckgo/Android/pull/4393/commits/7abd2b467982141303e21fff1dfad4585a46538b